Makefile

```diff
@@ -33,8 +33,9 @@ requirements-r:
 
 .PHONY: requirements-py
 requirements-py: .venv
-	$(VENV_BIN)/python -m pip install --upgrade pip
-	$(VENV_BIN)/pip install --upgrade mkdocs-material
+	python -m pip install --upgrade pip
+	pip install --upgrade mkdocs
+	pip install --upgrade mkdocs-material
```

Comment:
Because of [this error](https://github.com/pola-rs/r-polars/actions/runs/6989119884/job/19017929984#step:9:840) I had. This was just a patch to see if the rest works, I was planning to remove this change before merge